scripts/performance/perfcmp

```diff
@@ -0,0 +1,153 @@
+        elif field in self.TIME_FIELDS:
+            return '%-3.2f' % value
+        else:
+            return '%.2f' % value
+
+    def _human_readable_size(self, value):
+        hummanize_suffixes = ('KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB')
+        base = 1024
+        bytes_int = float(value)
+
+        if bytes_int == 1:
+            return '1 Byte'
+        elif bytes_int < base:
+            return '%d Bytes' % bytes_int
+
+        for i, suffix in enumerate(hummanize_suffixes):
+            unit = base ** (i+2)
+            if round((bytes_int / unit) * base) < base:
+                return ['%.2f' % (base * bytes_int / unit), suffix]
+
+    def diff_percent(self, field):
+        diff_percent = (
+            (self.new_summary[field] - self.old_summary[field]) /
+            float(self.old_summary[field])) * 100
+        return diff_percent
+
+
+def compare_runs(old_dir, new_dir):
+    for dirname in os.listdir(old_dir):
+        old_run_dir = os.path.join(old_dir, dirname)
+        new_run_dir = os.path.join(new_dir, dirname)
+        if not os.path.isdir(old_run_dir):
+            continue
+        old_summary = get_summary(old_run_dir)
+        new_summary = get_summary(new_run_dir)
+        comp = RunComparison(old_summary, new_summary)
+        header = [Style.BRIGHT + dirname + Style.RESET_ALL,
+                  Style.BRIGHT + 'old' + Style.RESET_ALL,
+                  # Numeric suffix (MiB, GiB, sec).
+                  '',
+                  'std_dev',
+                  Style.BRIGHT + 'new' + Style.RESET_ALL,
+                  # Numeric suffix (MiB, GiB, sec).
+                  '',
+                  'std_dev',
+                  Style.BRIGHT + 'delta' + Style.RESET_ALL]
+        rows = []
+        for field in comp.iter_field_names():
+            row = [field, comp.old(field), comp.old_suffix(field),
+                   comp.old_stddev(field), comp.new(field),
+                   comp.new_suffix(field), comp.new_stddev(field)]
+            diff_percent = comp.diff_percent(field)
+            diff_percent_str = '%.2f%%' % diff_percent
+            if diff_percent < 0:
+                diff_percent_str = (
+                    Fore.GREEN + diff_percent_str + Style.RESET_ALL)
+            else:
+                diff_percent_str = (
+                    Fore.RED + diff_percent_str + Style.RESET_ALL)
+            row.append(diff_percent_str)
+            rows.append(row)
+        print(tabulate(rows, headers=header, tablefmt='plain'))
+        print('')
+
+
+def get_summary(benchmark_dir):
+    summary_json = os.path.join(benchmark_dir, 'summary.json')
+    with open(summary_json) as f:
+        return json.load(f)
+
+
+def main():
+    parser = argparse.ArgumentParser(description='__doc__')
+    parser.add_argument('oldrunid', help='Path to old run idir')
+    parser.add_argument('newrunid', help='Local to new run dir')
+    parser.add_argument('--local', action='store_true')
```

Comment:
No I'll remove.  I originally wanted a local mode as well as a remote mode that would download from s3, but I've found that I just download the data from s3 and always have it local.  It might be worth adding the remote mode in the future, but running `s3 cp` ended up not being a big deal for me.